test-requirements.txt

```diff
@@ -4,6 +4,6 @@ koji
 configparser
 six > 1.10
 
-# Locked for python 2.7 support
+# Locked for python 2.6 support
 pytest-cov==2.5.1
-pytest==3.9.1
+pytest<3.3
```

Comment:
can you make this version lock conditional? Only for py 2.6 and a higher one for 2.7 or later? pytest 3 is already too old. If possible use latest 3.x or even 4.x for recent python.